utils/docker/Dockerfile.centos:7

```diff
@@ -10,36 +10,36 @@ FROM centos:7
 MAINTAINER Johann Lombardi <johann.lombardi@intel.com>
```

Comment:
Why does this file (with the `:` in the filename) still exist?

Reply:
Because Travis-CI will fail.
Need to create new files, merge them in, then change Travis-ci to use the new files.

Reply:
Is Travis using the `.travis.yml` from `master` on PR builds?

Reply:
It apparently is using it from the PR on PR builds.  If I remove the older filenames, it fails, so we can not convert Travis CI to the new names until this is merged to master and all in-flight PRs pick up the change.

Reply:
I'm not sure I'm following.  You say:

> It apparently is using it from the PR on PR builds.

Doesn't that mean that when you push this patch to Travis with the `:` named files removed, and the `.travis.yml` file updated to use the `.` names it will work as expected?

Reply:
I forgot completely about the .travis.yml file.
Fixed.